ui/src/app/completed/controllers/TableCtrl.js

```diff
@@ -6,20 +6,50 @@
 
   function TableCtrl($scope, ToastService, TableService, UtilityService) {
     $scope.downloads = [];
-
     var setSize = function(lst) {
       lst.data.forEach(function(download) {
         download.size = UtilityService.formatBytes(download.size);
       })
       return lst;
     };
-
+    $scope.isGridOn = false;
     TableService.getCompletedDownloads().then(function (response) {
       response = setSize(response);
       $scope.downloads = response.data;
     }, function(error){
       ToastService.showToast('Oops! Something went wrong fetching data');
     });
+    $scope.changeView = function () {
+        if($scope.isGridOn) {
+            document.getElementById('grid-button').innerText = 'view_module';
```

Comment:
can't we directly bind the value without directly manipulating the dom?

Reply:
like when I click the button, I have to change the inner text and I know only this way. If you have a good decision, please suggest 😊

Reply:
Can you explain the purpose of 'view_module' text? I actually didn't understand why it is been used. If you can explain it to me, maybe we could discuss the alternative solutions.

Reply:
actually, I want to change the icon of this button when someone clicks it.